cfpb/cfgov-refresh
# Email signup pop-up on owning-a-home/

Email signup pop-up on owning-a-home

Adding code for the email-signup popup on Owning a home.


## Testing

- Add the `MAIL-POPUP` feature flag.
- Visit `http://localhost:8000/owning-a-home/`.
- Scroll to the bottom of the screen.
- You should get the email signup pop-up.
- Enter your email address.
- Submit the form.
- Refresh your browser
- Scroll to the bottom.
- You shouldn't get the email popup this time.


## Screenshots

<img width="352" alt="screen shot 2017-02-27 at 6 29 28 pm" src="https://cloud.githubusercontent.com/assets/1696212/23384880/bcf85e1e-fd1a-11e6-9241-15325cb9a863.png">
<img width="632" alt="screen shot 2017-02-27 at 6 29 07 pm" src="https://cloud.githubusercontent.com/assets/1696212/23384881/bd0b1c66-fd1a-11e6-8fce-708a0204027d.png">

<img width="609" alt="screen shot 2017-02-27 at 6 32 17 pm" src="https://cloud.githubusercontent.com/assets/1696212/23384959/1b944942-fd1b-11e6-92c4-2480048b25c8.png">


## Todos

-

## Checklist
